tbx/project_styleguide/templates/patterns/molecules/streamfield/blocks/featured_services_block.html

```diff
@@ -1,22 +1,26 @@
 {% load wagtailcore_tags wagtailimages_tags %}
-<div>
+<div class="grid__featured-services {% if value.intro %}featured-services--with-intro{% endif %}">
     {# Section heading #}
-    <h2>{{ value.title }}</h2>
-
+    {% include "patterns/atoms/motif-heading/motif-heading.html" with heading_level=2 heading=value.title classes="motif-heading--two motif-heading--static featured-services__title " %}
     {# Section intro #}
-    <div>{{ value.intro|richtext }}</div>
-
-    {# Signposting cards #}
-    <ul>
+    {% if value.intro %}
+        <div class="text text--five featured-services__intro">{{ value.intro|richtext }}</div>
```

Comment:
we need to manually add `.rich-text` class for all the rich-text styles to show (links, lists, etc.)